### PR TITLE
Add authentication section to docs

### PR DESCRIPTION
**Pull request by neelxie**
## What does this PR do?

It adds the authentication part of docs to MKdocs

## Screenshot
image

### CODE REVIEW

File: cranecloud/docs/authentication/login.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Login
+1\. Browse this URL http://cranecloud.io/login to access the Crane Cloud login portal, similar to the one below
```
Comment (MutegekiHenry) [inline cranecloud/docs/authentication/login.md:2]: Kindly change the URL to the latest URL [<http://staging.cranecloud.io/login>](http://staging.cranecloud.io/login). you could make your links clickable by putting them in <> eg. <http://staging.cranecloud.io/login>.

File: cranecloud/docs/authentication/registration.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+## Registration
+1\. To register visit cranecloud.io and click the `GET STARTED `Button
```
Comment (MutegekiHenry) [inline cranecloud/docs/authentication/registration.md:2]: link changed to <http://staging.cranecloud.io/login>

File: cranecloud/docs/authentication/forgotPassword.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+## Forgot Password
+**User Actions:**
+
+Once you click the forgot password link on the login page, it will lead you to this route http://cranecloud.io/forgot-password that displays a page for you to enter your email address.
```
Comment (MutegekiHenry) [inline cranecloud/docs/authentication/forgotPassword.md:4]: link changed to <http://staging.cranecloud.io/login>

File: cranecloud/docs/authentication/login.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+![](../img/login.png)
+
+Enter your email address and password, then  click LOGIN Expected behavior:
```
[the remark this answers is not in the document]
Comment (neelxie) [reply cranecloud/docs/authentication/login.md]: @MutegekiHenry this has been worked on and fixed

File: cranecloud/docs/authentication/login.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+IF any field is left blank, the page throws an error saying “Please enter your email and password “ In this case, endeavor to have all the fields filled out and resubmit.
```
[the remark this answers is not in the document]
Comment (neelxie) [reply cranecloud/docs/authentication/login.md:9]: @MutegekiHenry this has been fixed and made bold

File: cranecloud/docs/authentication/login.md [hunk 6]
```diff
@@ -0,0 +1,11 @@
+## Login
+1\. Browse this URL http://cranecloud.io/login to access the Crane Cloud login portal, similar to the one below
+![](../img/login.png)
```
[the remark this answers is not in the document]
Comment (neelxie) [reply cranecloud/docs/authentication/login.md]: This image has been replaced @MutegekiHenry

File: cranecloud/docs/authentication/login.md (hunk 1 above)
Comment (neelxie) [reply cranecloud/docs/authentication/login.md]: This url has been fixed too @MutegekiHenry

File: cranecloud/docs/authentication/registration.md (hunk 2 above)
Comment (neelxie) [reply cranecloud/docs/authentication/registration.md]: This url too has been changed, @MutegekiHenry

File: cranecloud/docs/authentication/forgotPassword.md (hunk 3 above)
Comment (neelxie) [reply cranecloud/docs/authentication/forgotPassword.md:4]: This too is fixed now

**Pull request merged**